fortanix-vme/fortanix-vme-runner/src/lib.rs

```diff
@@ -12,6 +14,7 @@ use std::sync::{Arc, Mutex, RwLock};
 use fortanix_vme_abi::{self, Addr, Error as VmeError, Response, Request};
 use vsock::{self, SockAddr as VsockAddr, Std, Vsock, VsockListener, VsockStream};
 
+const MAX_MESSAGE_LEN: usize = 80;
```

Comment:
Nit: perhaps rename to `MAX_LOG_MESSAGE_LEN`? When I first saw this variable I thought it was the max length of some message communicated between the runner and the enclave.